src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const string = sourceString.split(';');
+
+  const result = {};
+
+  string.forEach((el) => {
+    const [key, value] = el.split(':').map((item) => item.trim());
```

Comment:
Add a check to ensure that `el.split(':')` results in exactly two elements. This will prevent errors if the input string is malformed or if there are missing keys or values.